src/Microsoft.Azure.WebJobs.Host/Timers/IWebJobsExceptionHandler.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.ExceptionServices;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Host.Timers
+{
+    /// <summary>
+    /// Represents an unhandled exception handler.
+    /// </summary>
+    public interface IWebJobsExceptionHandler
+    {
+        /// <summary>
+        /// Called during host creation. Allows a reference to the host.
+        /// </summary>
+        /// <param name="host">The JobHost.</param>
+        void Initialize(JobHost host);
```

Comment:
Why does this need to be part of the interface? I think instead a concrete implementation like WebJobsExceptionHandler  should just take the host as a ctor param, since its an implementation detail.